Assets/Project/Modules/PlayerAnchor/Scripts/Player/View/MaterialView/PlayerMaterialView.cs

```diff
@@ -90,8 +90,11 @@ public void PlayDashAnimation(float duration, Vector3 dashDirection)
 
         private async UniTaskVoid DoPlayDash(float duration, Vector3 dashDirection)
         {
+            _material.DOFloat(1.0f, _config.DashingProperty, _config.DashMaterialTransitionTime);
+            await UniTask.Delay(TimeSpan.FromSeconds(_config.DashMaterialTransitionTime));
             _rendererTransform.gameObject.SetActive(false);
-            await UniTask.Delay(TimeSpan.FromSeconds(duration));
+            await UniTask.Delay(TimeSpan.FromSeconds(duration - _config.DashMaterialTransitionTime));
```

Comment:
Hera also store the operation `duration - _config.DashMaterialTransitionTime` into a variable + apply  Max to clamp negative numbers